Update EP8 raster data cropping (#133)
carpentries-incubator/geospatial-python: _episodes/08-crop-raster-data.md

~~~
{: .output}

> As seen, the coordinate systems differ. To crop the raster using the shapefile, we first need to uniform the CRS. We can convert one to the other. But considering the size of `raster` is huge, to avoid loading the entire image, we will convert the coordinate system of `fields` to the coordinate system of `raster`:

Reviewer: ```suggestion
As seen, the coordinate systems differ. To crop the raster using the shapefile, we first need to reproject one dataset to the other's CRS. Since `raster` is large, we will convert the CRS of `fields` to the CRS of `raster` to avoid loading the entire image:
```

raster_clip_fields.plot.imshow(figsize=(8,8))
~~~
{: .language-python}
<img src="../fig/E08-04-crop-raster-crop-fields.png" title="Raster cropped by crop fields" width="512" style="display: block; margin: auto;" />

>  ## Challenge: crop raster data with a specific code

Author: ```suggestion
> ## Challenge: crop raster data with a specific code
```
